v2/invoke.go

```diff
@@ -83,6 +80,12 @@ func invoke(ctx context.Context, call APICall, settings CallSettings, sp sleeper
 		if strings.Contains(err.Error(), "x509: certificate signed by unknown authority") {
 			return err
 		}
+		if apierr, ok := FromError(err); ok {
+			return apierr
```

Comment:
Do not return here, we only want to set `err = apierr`. Returning here short circuits any retry that would happen.